app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_comment_counter
+
+  private
+
+  def update_comment_counter
+    post.update(comments_counter: post.comments.count)
```

Comment:
`post.increment!(:comments_counter)` would be a more efficient approach as you would not need to make extra db calls to run `post.comments.count`.